stdlib/CMakeLists.txt

```diff
@@ -1,4 +1,8 @@
 # Create convenience targets for the Swift standard library.
+
+set(CMAKE_CXX_COMPILER "${SWIFT_NATIVE_LLVM_TOOLS_PATH}/clang++")
+set(CMAKE_C_COMPILER "${SWIFT_NATIVE_LLVM_TOOLS_PATH}/clang")
```

Comment:
Please make this part optional through a `build-script-impl` option.  We have usecases for disabling it.  You can refer to c8d61fce9fc981719e9677a18cb44d3e0fc61a35 for an example of adding an option.